applications/trunkstore/src/ts_util.erl

```diff
@@ -92,6 +93,38 @@ constrain_weight(W) when W > 100 -> 100;
 constrain_weight(W) when W < 1 -> 1;
 constrain_weight(W) -> W.
 
+-spec lookup_trunk(kz_term:ne_binary(), kz_term:ne_binary()) ->
+                        {'ok', kz_json:object()} |
+                        {'error', 'no_trunk_found' | atom()}.
+lookup_trunk(TrunkId, AccountId) ->
+    AccountDb = kz_util:format_account_id(AccountId, 'encoded'),
+    case kz_cache:fetch_local(?CACHE_NAME
+                             ,{'lookup_trunk', TrunkId, AccountId}
+                             )
+    of
+        {'ok', _}=Resp ->
+            lager:info("cache hit for ~s", [TrunkId]),
+            Resp;
+        {'error', 'not_found'} ->
+            Options = [{'key', TrunkId}],
+            case kz_datamgr:get_results(AccountDb, ?TS_VIEW_CROSSBAR_LISTING, Options) of
```

Comment:
Why query the view? Why not just open the doc? The view's key is just `_id` anyway.

Reply:
To add, why is this function even needed? Nothing in trunkstore uses it, the doc is already cached via kz_datamgr.